admin/base/settings/defaults.py

```diff
@@ -20,6 +20,15 @@
 DEBUG = osf_settings.DEBUG_MODE
 DEBUG_PROPAGATE_EXCEPTIONS = True
 
+# secure mode for using https in local development
+SECURE = osf_settings.SECURE_MODE
+LOCAL = osf_settings.LOCAL_MODE
```

Comment:
These variables can probably be removed: `LOCAL` is unused, and `SECURE` makes more sense as a direct reference to the parent setting.